sql/mysqld.h

```diff
@@ -609,6 +609,7 @@ extern const char *mysql_real_data_home_ptr;
 extern ulong thread_handling;
 extern MYSQL_PLUGIN_IMPORT char  *mysql_data_home;
 extern "C" MYSQL_PLUGIN_IMPORT char server_version[SERVER_VERSION_LENGTH];
+extern "C" MYSQL_PLUGIN_IMPORT char server_version_suffix[SERVER_VERSION_LENGTH];
```

Comment:
Why it has to be extern "C" MYSQL_PLUGIN_IMPORT?

Reply:
Without it there is
```
CMakeFiles/handlersocket.dir/handlersocket/handlersocket.cpp.o:(.bss+0x0): multiple definition of `server_version_suffix'
```